bin/oxd

```diff
@@ -95,8 +96,16 @@ class LogFileStream(object):
         return True
 
     def log_file_path(self):
-        day_str = datetime.datetime.utcnow().strftime('%Y%m%d')
-        return os.path.normpath(os.path.join(self.log_path + '/', "%s/%s-%s.log" % (day_str, self.name, day_str)))
+        now = datetime.datetime.utcnow()
+        day_str = now.strftime('%Y%m%d')
+        if self.rotate_hours:
+            hour = now.hour - (now.hour % self.rotate_hours)
```

Comment:
Maybe my brain is broken but if `self.rotate_hours > now.hour`, `hour_str` will always be zero is that correct?